build.gradle

```diff
@@ -92,10 +93,9 @@ subprojects {
             }
         }
     }
-}
 
 def excludeProjects() {
-    subprojects.findAll { project -> (project.name != 'proto' && project.name != 'osgi-wrappers')}
+    project.name != 'proto' && project.name != 'osgi-wrappers'
```

Comment:
Looking at where `excludeProjects()` is used, this method signature should be `Collection<Project> excludeProjects() {`

The change here changed the return value from `Collection<Project>` to `boolean`. Not sure how that's supposed to work.